### PR TITLE
Ports dildos

### PR DESCRIPTION
**Pull request by Spacedong**
ports dragon dildos by nexendia back into the game

currently only adminspawnable sorry

### CODE REVIEW

**Comment by ArcLumin** [comment]
:HAPPENING:

**Comment by Memendia** [comment]
Fuck off

**Comment by Swagile** [comment]
where the meme tag at????

**Comment by Spacedong** [comment]
hold on ill add embedchance

**Comment by GuyonBroadway** [comment]
Pls add a dildo made of salt, call it the salt-dong.

**Comment by ArcLumin** [comment]
git it. you can't reproduce with a dildo

I'm so funny

**Comment by GuyonBroadway** [comment]
We need a tag called [Rainbow of Tags]... or, as tags are a refraction of the spectrum of visible light the tag should be [A Spectrum of tags and other learning difficulties]

**Comment by ArcLumin** [comment]
>not being able to embed yourself on the shelf of dildos

**Comment by GuyonBroadway** [comment]
Make it so you can add dildoes to unfinished meatspike frames and them climb onto them.

**Comment by Iamgoofball** [comment]
@Memendia you literally made these

File: html/changelogs/Spacedong - dildos.yml [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+#   rscdel (general deleting of nice things)
+#   imageadd
+#   imagedel
+#   spellcheck (typo fixes)
+#   experiment
+#   tgs (TG-ported fixes?)
+#################################
+
+# Your name.  
+author: Spacedong
```
Comment (Iamgoofball) [inline html/changelogs/Spacedong - dildos.yml:25]: should correct this to Nexendia

**Comment by ArcLumin** [comment]
meme-in-dia's children came back to fuck em

File: html/changelogs/Spacedong - dildos.yml (hunk 1 above)
Comment (Spacedong) [inline html/changelogs/Spacedong - dildos.yml]: its a port you nigger

Comment (AliiTea) [reply html/changelogs/Spacedong - dildos.yml:25]: Give me credit!

Comment (Spacedong) [inline html/changelogs/Spacedong - dildos.yml]: fine nigger

Comment (ArcLumin) [reply html/changelogs/Spacedong - dildos.yml:25]: it's a nigger you port

**Comment by ArcLumin** [comment]
Saltdong dido when

literally adds sodium to anyone who touches it

**Comment by Spacedong** [comment]
also you forgot [needs code review]

**Comment by Spacedong** [comment]
@ArcLumin

**Comment by Spacedong** [comment]
let the +1's pour in

**Comment by TDcoolguy300** [comment]
-1, no polluting my christian station with the tools of the devil.

**Comment by ArcLumin** [comment]
Kosherstation 13 lads, these dildos need to be circumcised

**Comment by Zadeon** [comment]
Uhh no

**Comment by yobob591** [comment]
+10000000000000000000000000000000000000 merge now

File: code/game/objects/items/dragon.dm [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+	icon = 'icons/obj/dicks.dmi'
+	icon_state = null
+	force = 5
+	w_class = 1
+	throwforce = 5
+	hitsound = 'sound/items/squishy.ogg'
+	attack_verb = list("slapped")
+
+/obj/item/dragon/suicide_act(mob/user)
+	user.visible_message(pick("<span class='suicide'>[user] is shoving [src.name] down \his throat! It looks like they're trying to commit suicide.</span>"))
```
Comment (JohnGinnane) [inline code/game/objects/items/dragon.dm:15]: Should probably check gender for "his"/"her"

Comment (ArcLumin) [reply code/game/objects/items/dragon.dm:15]: did you just

assume

their gender


also we have 3 genders in our code git it rite

**Comment by TDcoolguy300** [comment]
They must follow by MY gender roles, otherwise they are banned forever.

**Comment by ArcLumin** [comment]
WEW

**Comment by banthebantz** [comment]
This has summoned goofball, I'm not sure if that is a good or a bad omen

**Comment by Pyko1** [comment]
We don't need dildos, but saltdong should be added. 
1000 units of NaCl to make massive dong statue.

File: code/game/objects/items/dragon.dm (hunk 2 above)
Comment (Pyko1) [inline code/game/objects/items/dragon.dm]: pretty sure /his actually does gender check and adapt according to gender.

Comment (Pyko1) [inline code/game/objects/items/dragon.dm]: ie.
```
/obj/item/weapon/scalpel/suicide_act(mob/user)
	user.visible_message(pick("<span class='suicide'>[user] is slitting \his wrists with [src]! It looks like \he's trying to commit suicide.</span>", \
						"<span class='suicide'>[user] is slitting \his throat with [src]! It looks like \he's trying to commit suicide.</span>", \
						"<span class='suicide'>[user] is slitting \his stomach open with [src]! It looks like \he's trying to commit seppuku.</span>"))
```

Comment (JohnGinnane) [reply code/game/objects/items/dragon.dm:15]: Oh, that's pretty neat

**Comment by JohnGinnane** [comment]
Code looks good

**Comment by McBawbaggings** [comment]
Descriptions are terrible.
Also not particularly kosher :-DDD

**Comment by Pyko1** [comment]
Also horse cock dildo wont fit in a regular butt!

**Comment by McBawbaggings** [comment]
LOL @ PR NAME

**Comment by Swagile** [comment]
i will only +1 this meme pr if, and only if, anyone with a dick is valid by crew because they are dicks and the only weapon that those with dicks have to defend themselves with is shooting cum at people like a fire hydrant

if not, this is not meme enough for me to +1 and shall be removed !!!!!!

**Comment by McBawbaggings** [comment]
Descriptions are not terrible, totally not because of me.
Although in seriousness, I feel as if the butt stuffing thing should have an object type blacklist for /obj/item/dragon

**Comment by Spacedong** [comment]
i fucking credited you you nigger

**Comment by McBawbaggings** [comment]
I was joking ya nig

**Comment by Spacedong** [comment]
dickz

**Comment by zenek123321** [comment]
+1

**Comment by yobob591** [comment]
the people have spoken, merge dilods headmins

**Comment by TDcoolguy300** [comment]
Nah

**Comment by yobob591** [comment]
being serious as an admin spawned thing this would be fun for meme events

**Comment by TDcoolguy300** [comment]
No

**Comment by Dudeinyourbackyard** [comment]
nnnnnnoooooooooo

**Comment by Pyko1** [comment]
No RHVD2BD?

aka Rapid High Velocity Dildo to Butt Deployer.

**Comment by Hackole** [comment]
gross -1

**Comment by calibraptor** [comment]
Do it

**Comment by yobob591** [comment]
Im waiting

**Comment by doobiedoo23** [comment]
Fix the descriptions retard, they still have Nex's shitty ones that had to be fixed last time

**Comment by doobiedoo23** [comment]
Disregard that I suck cocks

**Comment by JohnGinnane** [comment]
@McBawbaggings are you approving this? Hackolé has -1 so I'm not sure where we're at..

**Comment by McBawbaggings** [comment]
If butts have a blacklist for dildo type objects, I would be okay with it since they're only admin spawnable at this point.

**Comment by JohnGinnane** [comment]
You want snowflake code to exclude dildos from being put in people's asses?

**Comment by McBawbaggings** [comment]
I was moreso thinking to have a var for all objects "smugglable" or some shit. It defaults to 1 but you can set it to 0 for certain objects.

**Comment by JohnGinnane** [comment]
Ah, that makes way more sense. Technically you can only put small/medium things up there, so we could just make the dildos really big so they won't fit?

**Comment by McBawbaggings** [comment]
Bluespace butts can take normal size items. I think making a var might be a bit better rather than balancing items to be unable to fit inside butts. Dildos being too big to put into backpacks is weird.

**Comment by JohnGinnane** [comment]
Butts have the same size as backpacks? If we could find something that's not in use or something instead of creating a new var on an object like /obj/, that means more memory just to see if you can put something in your ass. Would be neat if we could collate all these simple 1/0 variables so we don't have to waste memory

**Comment by Iamgoofball** [comment]
Am I reading this thread right 

Is this the real life
Is this just fantasy

**Comment by Carbonhell** [comment]
fuck you all
+1ing this as long as you fix whatever's broken

**Comment by ArcLumin** [comment]
Just make dildos the child of organs. Organs can't be shoved up your butt

also, it means you can eat dildos

**Comment by Carbonhell** [comment]
Actually yeah,do what arc said

**Comment by JohnGinnane** [comment]
Does that mean you could replace someone's arms and legs with dildos?

**Comment by ArcLumin** [comment]
If you put it in the right slot, maybe?

Best to put it in the crotch slot though :^)

**Comment by McBawbaggings** [comment]
>Force feeding people dildos
Replacing someone's appendix/heart with a dildo

**Comment by ArcLumin** [comment]
>find someone who pissed off an admin
>open up their body
>hundreds of dildos inside

**Comment by Spacedong** [comment]
what have i done

**Comment by Spacedong** [comment]
I just noticed that one of the tags was [Needs Head Insults] instead of [Needs Head Input] just now

**Comment by TDcoolguy300** [comment]
No

**Comment by JohnGinnane** [comment]
@Hackole @McBawbaggings thoughts on this?

**Comment by SplergeBucket** [comment]
why are there so many comments over an object im going to use to embed in someone's skull
ok

**Comment by ArcLumin** [comment]
@spacedong this is good to merge once you fix the issues. Namely, make dildos organs.

**Comment by JohnGinnane** [comment]
@Spacedong are you gonna work on this?

**Comment by Nebyab** [comment]
yes

**Comment by SpankMasterson** [comment]
ehhh i dont like but ill do it if its what the people want, i vote whatever mcbaw votes

**Comment by yobob591** [comment]
+111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111111

**Comment by Swagile** [comment]
was this fully code checked?

im just asking because i don't want another pizza mob port type incident

**Comment by McBawbaggings** [comment]
Okay, now it's buttproof. Gigantic dildo is kinda funny anyway.

**Comment by Spacedong** [comment]
WEW

**Pull request merged**